packages/aws-cdk-lib/aws-docdb/lib/ca-certificate.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * The CA certificate used for a DB instance.
+ *
+ * @see https://docs.aws.amazon.com/documentdb/latest/developerguide/ca_cert_rotation.html
+ */
+export class CaCertificate {
+  /**
+   * rds-ca-2019 certificate authority
+   */
+  public static readonly RDS_CA_2019 = new CaCertificate('rds-ca-2019');
+
+  /**
+   * rds-ca-rsa2048-g1 certificate authority
+   */
+  public static readonly RDS_CA_RSA2048_G1 = new CaCertificate('rds-ca-rsa2048-g1');
+
+  /**
+   * rds-ca-rsa4096-g1 certificate authority
+   */
+  public static readonly RDS_CA_RSA4096_G1 = new CaCertificate('rds-ca-rsa4096-g1');
+
+  /**
+   * @param identifier - CA certificate identifier
+   */
+  private constructor(private readonly identifier: string) { }
+
```

Comment:
I see RDS has the following:

```
  /**
   * Custom CA certificate
   *
   * @param identifier - CA certificate identifier
   */
  public static of(identifier: string) {
    return new CaCertificate(identifier);
  }
```

Do we need to support this? If not, this could be an `enum`.

Reply:
That's a good question!

#27138 allows for it, so we probably should too for consistency. Anyway, in case a new CA gets added, it is probably better to have this instead of relying on a property override?

I've added the method.

Reply:
Possible to add a unit and integration test for this?